### DISCUSSION TITLE
Wrong content type, page download instead of display

### DISCUSSION THREAD

**Discussion by andreg** [vercel/community#4728] [Bug Report]
### Page to Investigate

https://advancedcolumns-site-bec3uhxuu-evolvesnc.vercel.app/changelog

### Steps to Reproduce

For some reason, when accessing the page, we get an `application/octet-stream` content type in return, which results in the browser attempting to download it.

We are building a static site, running on a pretty standard Eleventy.

Why is this happening? Thank you!

**Comment by amyegan** [comment]
Hi @andreg. Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that I can use to help with debugging? I tried to reproduce the problem using the ["getting started" example](https://www.11ty.dev/docs/get-started/), but everything deployed correctly for me.
